h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -479,8 +489,17 @@ public void init(boolean expensive) {
           _state._ymu = MemoryManager.malloc8d(_nclass);
           for (int i = 0; i < _state._ymu.length; ++i)
             _state._ymu[i] = _priorClassDist[i];
-        } else
-          _state._ymu = new double[]{_parms._intercept?_train.lastVec().mean():_parms.linkInv(0)};
+        } else {
+          _state._ymu = new double[]{_parms._intercept ? _train.lastVec().mean() : _parms.linkInv(0)};
+/*          if (_parms._family.equals(Family.negativebinomial) && _parms._optimize_theta) {
+            double rmean = _train.lastVec().mean();
+            rmean = rmean>0?rmean:eps;
+            double rstd = _train.lastVec().sigma();
+            double temp = (rstd*rstd-rmean)/(rmean*rmean);
+            _parms._theta = temp>0?temp:eps;
+            _parms._invTheta = 1/_parms._theta;
+          }*/
+        }
```

Comment:
please ignore all parts concerning theta optimization.  I am disabling them for now but will like to add them back later.

Reply:
Done